graphql/queries/health.graphql

```diff
@@ -0,0 +1,3 @@
+query HealthQuery {
```

Comment:
All queries (and mutations/subscriptions) are placed in `.graphql` files, which are validated at compile time and embedded in the bin as strings.